### DISCUSSION TITLE
Geo-distributed teleport

### DISCUSSION THREAD

**Discussion by gordonswing** [gravitational/teleport#14573] [Q&A]
Hello there!
I'm looking for a solution for running one (or several) geo-distributed cluster to cover one important thing: my servers and k8s clusters running in LATAM, US, EU, and APAC, using one teleport cluster in one region is painful cause to connect to k8s cluster in the US from EU (where I'm right now) I should create a connection to my teleport cluster in APAC. It increases latency and I cannot work with that latency well.
First point: I thought about one three-node cluster - one node in the US, the second in the EU and the third in APAC. Is it a good idea?
Second point: how I can force users to create connections to resources from the exact node? For example, US engineers use only the US node for going anywhere.

**Comment by Valien** [comment]
Hi @gordonswing -

You can leverage Trusted Clusters and stand up a cluster in each region and access them from a root cluster (https://goteleport.com/docs/setup/admin/trustedclusters/).  

And you can leverage RBAC roles to limit what users can see, so US users can only login to the US cluster and see those resources, etc.
